NorthstarDLL/modmanager.cpp

```diff
@@ -369,6 +500,14 @@ void ModManager::LoadMods()
 															 // behaviour is for defining same convar multiple times
 				new ConVar(convar->Name.c_str(), convar->DefaultValue.c_str(), convar->Flags, convar->HelpString.c_str());
 
+		for (ModConCommand* command : mod.ConCommands)
```

Comment:
The mix of no braces on the for and braces on the if really don't go well, maybe add braces?